curriculum/challenges/english/25-front-end-development/quiz-responsive-web-design/66ed9034f45ce3ece4053ebd.md

```diff
@@ -523,3 +523,444 @@ What is the default media type if no media type is specified?
 
 `all`
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which CSS unit is commonly used in responsive design to create fluid layouts?
+
+#### --distractors--
+
+`px`
+
+---
+
+`em`
+
+---
+
+`rem`
+
+#### --answer--
+
+`%`
+
+### --question--
+
+#### --text--
+
+What is the primary purpose of using media queries in responsive web design?
+
+#### --distractors--
+
+To apply different styles based on user input
+
+---
+
+To make the website load faster
+
+---
+
+To change the website’s color scheme automatically
+
+#### --answer--
+
+To adjust the layout and design based on screen size
+
+### --question--
+
+#### --text--
+
+Which media feature can be used to detect if a user prefers reduced motion for accessibility reasons?
+
+#### --distractors--
+
+`motion-preference`
+
+---
+
+`reduce-motion`
+
+---
+
+`min-motion`
+
+#### --answer--
+
+`prefers-reduced-motion`
```

Comment:
This topic is taught in the CSS Animations section that comes later in the course. Kindly replace this question with a topic from the current section (Responsive Webdesign) only.